### PR TITLE
updated Az.Accounts Version

### PR DESCRIPTION
**Pull request by abbhar**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by wyunchi-ms — APPROVED** [review]

File: tools/Az/Az.psd1 [hunk 1]
```diff
@@ -53,7 +53,7 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.1.2'; },
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.2.4'; },
```
Comment (bganapa) [inline tools/Az/Az.psd1:56]: @abbhar we need this change in all the modules psd1 files. Lets modify all the psd1 files. we will find a better way next time

**Review by bganapa — CHANGES_REQUESTED** [review]
comments

**Comment by rakku-ms** [comment]
@bganapa do we need  Az.Accounts version update in src/Accounts/Accounts/Az.Accounts.psd1 line 15 (from 2.0.1 to 2.2.4)?

**Comment by abbhar** [comment]
> @bganapa do we need Az.Accounts version update in src/Accounts/Accounts/Az.Accounts.psd1 line 15 (from 2.0.1 to 2.2.4)?
Updating it, to keep it consistent.

**Review by bganapa — APPROVED** [review]

**Pull request merged**